src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/extensions/system_capability_app_extension.cc

```diff
@@ -0,0 +1,97 @@
+#include "sdl_rpc_plugin/extensions/system_capability_app_extension.h"
+
+namespace sdl_rpc_plugin {
+CREATE_LOGGERPTR_GLOBAL(logger_, "GetSystemCapabilitiesAppExtension")
+
+namespace app_mngr_ = application_manager;
+unsigned SystemCapabilityAppExtension::SystemCapabilityAppExtensionUID = 200;
+
+SystemCapabilityAppExtension::SystemCapabilityAppExtension(
+    sdl_rpc_plugin::SDLRPCPlugin& plugin, app_mngr_::Application& app)
+    : app_mngr_::AppExtension(
+          SystemCapabilityAppExtension::SystemCapabilityAppExtensionUID)
+    , plugin_(plugin)
+    , app_(app) {}
+
+SystemCapabilityAppExtension::~SystemCapabilityAppExtension() {}
+
+bool SystemCapabilityAppExtension::subscribeTo(
```

Comment:
Class functions should be named with `UpperCamelCase`: https://google.github.io/styleguide/cppguide.html#Function_Names